src/Athena/Controllers/HomeController.cs

```diff
@@ -4,28 +4,43 @@
 
 namespace Athena.Controllers
 {
-    [Route("[controller]")]
+    [Route("")]
     public class HomeController : Controller
     {
-        [HttpGet("")]
-        public IActionResult Index()
+        [HttpGet("Home")]
+        [HttpGet]
+        public IActionResult Home()
         {
             return View();
         }
 
-        [HttpGet("About")]
-        public IActionResult About()
+        [HttpGet("Login_Page")]
+        public IActionResult Login_Page()
         {
-            ViewData["Message"] = "Your application description page.";
+            return View();
+        }
+
+        [HttpGet("Add_Classes")]
+        public IActionResult Add_Classes()
```

Comment:
This will need moved to a separate controller eventually. I am OK with leaving this here for now if you want to ease view development but it's something to keep in mind.